jwala-services/src/test/java/com/cerner/jwala/service/app/impl/ApplicationServiceImplTest.java

```diff
@@ -91,7 +100,13 @@ public void setUp() {
         groupId2 = new Identifier<Group>(2L);
         group = new Group(groupId, "the-ws-group-name");
         group2 = new Group(groupId2, "the-ws-group-name-2");
-
+        try {
+            PowerMockito.mockStatic(JwalaUtils.class);
+            PowerMockito.when(JwalaUtils.getHostAddress("testServer")).thenReturn(Inet4Address.getLocalHost().getHostAddress());
+            PowerMockito.when(JwalaUtils.getHostAddress("testServer2")).thenReturn(Inet4Address.getLocalHost().getHostAddress());
+        }catch (UnknownHostException ex){
```

Comment:
Instead of catching the UnknownHostException we can just do this which looks cleaner. The stacktrace will get displayed by JUnit when there's a problem.

    public void setUp() throws UnknownHostException {
        ...
    }